CHANGELOG.md

```diff
@@ -17,6 +17,17 @@ The REST API versions in the table are just for your information as the values a
 
 - We ask developers to consider using the Headline component instead of the KPI component. The KPI component may be eventually marked as deprecated in one of the next major versions.
 
+## 7.5.0
+
+### Added
+
+- Geo charts (pushpin) component ([doc](https://sdk.gooddata.com/gooddata-ui/docs/geo_pushpin_chart_component.html))
+- Measure value filter dropdown with button component in SDK
+
+### Fixed
+
+- Visualization rendering after error that persisted even though there was no longer an error
```

Comment:
- Visualization rendering stuck due to an error that persisted after the error was gone